packages/patternfly-3/patternfly-react/src/components/HorizontalNav/HorizontalNavMenuItem.js

```diff
@@ -4,23 +4,23 @@ import classNames from 'classnames';
 import { Dropdown, ListGroup, ListGroupItem } from '../../index';
 
 const HorizontalNavMenuItem = props => {
-  const { children, onItemClick, title, active, dropdown, submenu, dropup, pullLeft } = props;
+  const { children, onItemClick, title, active, dropdown, submenu, dropup, pullLeft, ...otherProps } = props;
 
   const dropdownClasses = classNames({
     'dropdown-submenu': submenu,
     'pull-left': pullLeft
   });
 
   return dropdown ? (
-    <Dropdown componentClass="li" className={dropdownClasses}>
-      <a bsRole="toggle">
+    <Dropdown componentClass="li" className={dropdownClasses} {...otherProps}>
+      <Dropdown.Anchor bsRole="toggle">
```

Comment:
What about using <Dropdown.Toggle useAnchor={true} />?

Reply:
Better! Didn't see that option as it's not in the docs at https://react-bootstrap.github.io/components/dropdowns/